docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -101,15 +101,13 @@ As an alternative to using the {product-kafka} web console, you can use the `rho
 rhoas kafka topic update --name my-kafka-topic --retention-ms 704800000
 ----
 
-For a list of topic properties that you can update using the CLI, see the `rhoas kafka topic update` entry in the {base-url-cli}{command-ref-url-cli}[_CLI command reference (rhoas)_^].
+For a list of topic properties that you can update using the CLI, see the `rhoas kafka topic update` entry in the {base-url-cli}{command-ref-url-cli}[CLI command reference (rhoas)^].
 
 .Prerequisites
-* You are logged in to the {product-kafka} web console.
-* You have created a Kafka instance with at least one Kafka topic in {product-kafka}.
+* You have created a Kafka instance with at least one Kafka topic in {product-kafka}. To learn how to do this, see {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^].
```

Comment:
@jbyrne-redhat Only reason I added that cross ref was that Steve did so in the schema sections at the end, which merits them given the different context. For consistency, I've added the same here. Also for consistency, I used his same wording "To learn more" instead of what I normally use ("For more information ..." ) I think Steve was trying to avoid the repetition of "doing x, y, z.  For more information about doing x, y, z, see ...."  I typically try to reword, but for now just stuck with his alternative, which works.

Reply:
Well that's handy. Ack.